bin/GameIndex.dbf

```diff
@@ -12344,6 +12369,11 @@ Compat = 5
 Serial = SLES-52711
 Name   = Titeuf Mega Compet
 Region = PAL-F
+[patches = 958E5086]
+	comment= patched by Prafull
+	//Sheep screen freeze fix
```

Comment:
Add a space after `//`.
I assume Sheep here is literal, if not remove the word.